README.md

```diff
@@ -1,2 +1,20 @@
-# MAX-Framework
- Python package that can be installed to make it easier to create MAX models
+# Model Asset Exchange Framework
+ Python package that contains common code shared across all MAX models.
+
+## Dependencies
+* [flask-restplus](https://pypi.org/project/flask-restplus/0.11.0/)
+* [flask-cors](https://pypi.org/project/Flask-Cors/)
+
+## Installation
+The package can be installed with pip. However, this is not necessary as the
+Docker images of all MAX models will include `maxfw` as a package.
```

Comment:
Maybe change this to something like:
```
The package can be installed with pip. However, this is not necessary as each MAX model will get the `maxfw` package via the `MAX-Base` image`.

Reply:
Thanks for the suggestion, I think is a better way to put it.